crud/src/main/java/com/redhat/lightblue/crud/CRUDController.java

```diff
@@ -125,4 +120,9 @@ CRUDFindResponse find(CRUDOperationContext ctx,
      * The back end should update the predefined fields of the document
      */
     void updatePredefinedFields(CRUDOperationContext ctx, JsonDoc doc);
+    
+    /**
+     * Checks the CRUD health
```

Comment:
Would below comment be suitable?

"Checks the CRUD controller health. The respective implementations should provide the health of lightblue CRUD layer which is further exposed to external systems by dropwizard metrics HealthCheckServlet REST endpoint"